.gitlab-ci.yml

```diff
@@ -218,6 +207,90 @@ build_windows_msi_x64:
     paths:
     - .omnibus/pkg
 
+
+
+#
+# image_build
+#
+
+
+# build the agent6 image
+build_agent6:
+  stage: image_build
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/devtools/dind/build:latest
+  tags: [ "runner:dind", "size:large" ]
+  script: [ "# noop" ]
+  variables:
+    DD_DIND_BUILD_CONTEXT: Dockerfiles/agent6
+    DD_DIND_IMAGE: 486234852809.dkr.ecr.us-east-1.amazonaws.com/datadog-agent/agent:$CI_COMMIT_SHA
```

Comment:
nit: we tend to use `$CI_PIPELINE_ID` for temporary tags, as they can be sorted (for cleanup) and linked to a pipeline directly

Reply:
@adrienkohlbecker I can totally change this, I was unsure what was the most convenient thing to use, because using the pipeline_id you have one extra step to see what was the commit linked to it

Reply:
I'll change this to `$CI_PIPELINE_ID`